tools/slang-capability-generator/capability-generator-main.cpp

```diff
@@ -45,13 +52,18 @@ struct SerializedArrayView
 
 struct CapabilityDef : public RefObject
```

Comment:
the only closed variable is `abstractAtomsPresent`. Once it is properly encapsulated, things should be fine.

Reply:
Your implementation indeed encapsulated the variable `abstractAtomsPresent` but the struct/class is now in a strange state that is neither.
This is not good from the class design perspective.
I don't know what you mean by "Once".
And I don't think it will be "fine" from the design perspective.

Reply:
I will just make it public